layout/src/markup/projects/repository.hbs

```diff
@@ -4,6 +4,6 @@
     <p>{{description}}</p>
   </main>
   <footer>
-    <span>{{lang}}</span>
+    <span class="{{class}}">{{lang}}</span>
```

Comment:
When you refactor this will go away! And will be static given the `lang` attribute!